src/gluonts/mx/trainer/_base.py

```diff
@@ -98,6 +98,12 @@ class Trainer:
     init
         Initializer of the weights of the network (default: "xavier").
     hybridize
+        If set to true the network will be hybridized before training
+    after_initialize_cb
```

Comment:
Maybe `post_initialize` is a nicer name? Personally, I'm not a fan of the `_cb` prefix.